helm-chart/templates/nextcloud-aio-talk-deployment.yaml

```diff
@@ -0,0 +1,48 @@
+{{- if eq .Values.TALK_ENABLED "yes" }}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  annotations:
+    kompose.cmd: kompose convert -c -f latest.yml
+    kompose.version: 1.27.0 (b0ed6a2c9)
+  labels:
+    io.kompose.service: nextcloud-aio-talk
+  name: nextcloud-aio-talk
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      io.kompose.service: nextcloud-aio-talk
+  strategy: {}
```

Comment:
Do we always need a strategy?

Reply:
No. I recommend omitting it, as I'm not sure if the current version is valid (for Kubernetes). The strategy defaults to `RollingUpdate` which means replacing pods one by one. 

Again, as there is only a single replica, there is not much difference to the alternative strategy `Recreate`.

Reply:
removed strategies